Migrations/update4.recipe.json

```diff
@@ -0,0 +1,84 @@
+{
+  "steps": [
+    {
+      "name": "ContentDefinition",
+      "ContentTypes": [
+        {
+          "Name": "NotificationMenuItem",
+          "DisplayName": "NotificationMenuItem",
+          "Settings": {
+            "ContentTypeSettings": {
+              "Stereotype": "MenuItem"
+            },
+            "FullTextAspectSettings": {}
+          },
+          "ContentTypePartDefinitionRecords": [
+            {
+              "PartName": "NotificationMenuItem",
+              "Name": "NotificationMenuItem",
+              "Settings": {
+                "ContentTypePartSettings": {
+                  "Position": "1"
+                }
+              }
+            },
+            {
+              "PartName": "TitlePart",
+              "Name": "TitlePart",
+              "Settings": {
+                "ContentTypePartSettings": {
+                  "Position": "0"
+                }
+              }
+            },
+            {
+              "PartName": "LinkVisualPart",
+              "Name": "LinkVisualPart",
+              "Settings": {
+                "ContentTypePartSettings": {
+                  "Position": "2"
+                }
+              }
+            }
+          ]
+        }
+      ],

```

Comment:
The LinkVisualPart being on this creates an interesting scenario if they try to turn it into e.g a button... Not anything you need to fix, just an observation, thinking out loud :)

Reply:
The theme will need to cater for the fact that `notification` could be a button.